shannon/mainnet/config.toml

```
@@ -211,13 +211,13 @@ laddr = "tcp://0.0.0.0:26656"
 external_address = ""
 
 # Comma separated list of seed nodes to connect to
-seeds = "b5b11240b89c9fb86fb8dc78915d454fa0527465@shannon-grove-seed1.mainnet.poktroll.com:26656"
+seeds = ""
```

Let's keep the seeds here. We want other participants to use the seeds.

```suggestion
seeds = "b5b11240b89c9fb86fb8dc78915d454fa0527465@shannon-grove-seed1.mainnet.poktroll.com:26656"
```